### ISSUE TITLE
Perl warning

### ISSUE THREAD

**Issue by ioguix** [dalibo/PAF#39]
There's a perl warning in logs when score does not exists yet.

Around line 614, in `check_location`, the sub `_get_master_score` return an empty string when score does not exists. This raise the warning on next tests related to `$node_score`:

```
_set_master_score( -1, $row->[0] ) if $node_score != -1;
[...]
if ( $node_score != $row->[1] ) {
```

Same issue than in e4d98a4.

**Comment by ioguix** [comment]
Fixed in 7c3340a88f35c9ff2b40dde412d4e69c27680c3e

**Comment by ioguix** [comment]
This needs to be fixed in v1.1 as well. So I reopen the issue.

**Comment by ioguix** [comment]
fixed in fece017
